modules/cudl-data-processing/s3.tf

```diff
@@ -108,32 +108,68 @@ resource "aws_s3_bucket_notification" "source-bucket-notifications" {
 }*/
 
-// TODO also temp. hack to get multiple triggers working on a single bucket.
-resource "aws_s3_bucket_notification" "dest-bucket-notifications" {
-  #count  = length(var.db-lambda-information)
-  bucket = aws_s3_bucket.dest-bucket.id
-
-  queue {
-    queue_arn     = aws_sqs_queue.db-lambda-sqs-queue[0].arn
-    events        = ["s3:ObjectCreated:*", "s3:ObjectRemoved:*"]
-    filter_prefix = try(var.db-lambda-information[0].filter_prefix, "") != "" ? var.db-lambda-information[0].filter_prefix : null
-    filter_suffix = try(var.db-lambda-information[0].filter_suffix, "") != "" ? var.db-lambda-information[0].filter_suffix : null
-  }
-
-  queue {
-    queue_arn     = aws_sqs_queue.db-lambda-sqs-queue[1].arn
-    events        = ["s3:ObjectCreated:*", "s3:ObjectRemoved:*"]
-    filter_prefix = try(var.db-lambda-information[1].filter_prefix, "") != "" ? var.db-lambda-information[1].filter_prefix : null
-    filter_suffix = try(var.db-lambda-information[1].filter_suffix, "") != "" ? var.db-lambda-information[1].filter_suffix : null
-  }
-
-  queue {
-    queue_arn     = aws_sqs_queue.db-lambda-sqs-queue[2].arn
-    events        = ["s3:ObjectCreated:*", "s3:ObjectRemoved:*"]
-    filter_prefix = try(var.db-lambda-information[2].filter_prefix, "") != "" ? var.db-lambda-information[2].filter_prefix : null
-    filter_suffix = try(var.db-lambda-information[2].filter_suffix, "") != "" ? var.db-lambda-information[2].filter_suffix : null
-  }
-  # without the `depends_on` argument, the bucket notification creation fails because the
-  # lambda function doesn't exist yet
-  depends_on = [aws_lambda_function.create-db-lambda-function]
-}
+# // TODO also temp. hack to get multiple triggers working on a single bucket.
+# resource "aws_s3_bucket_notification" "dest-bucket-notifications" {
+#   #count  = length(var.db-lambda-information)
+#   bucket = aws_s3_bucket.dest-bucket.id
+#
+#   dynamic "queue" {
+#     for_each = aws_sqs_queue.db-lambda-sqs-queue
+#     content {
+#       queue_arn     = aws_sqs_queue.db-lambda-sqs-queue[each.key].arn
+#       events        = ["s3:ObjectCreated:*", "s3:ObjectRemoved:*"]
+#       filter_prefix = try(var.db-lambda-information[0].filter_prefix, "") != "" ?
+#         var.db-lambda-information[0].filter_prefix : null
+#       filter_suffix = try(var.db-lambda-information[0].filter_suffix, "") != "" ?
+#         var.db-lambda-information[0].filter_suffix : null
+#     }
+#   }
+#
+#   queue {
+#     queue_arn     = aws_sqs_queue.db-lambda-sqs-queue[1].arn
+#     events        = ["s3:ObjectCreated:*", "s3:ObjectRemoved:*"]
+#     filter_prefix = try(var.db-lambda-information[1].filter_prefix, "") != "" ? var.db-lambda-information[1].filter_prefix : null
+#     filter_suffix = try(var.db-lambda-information[1].filter_suffix, "") != "" ? var.db-lambda-information[1].filter_suffix : null
+#   }
+#
+#   queue {
+#     queue_arn     = aws_sqs_queue.db-lambda-sqs-queue[2].arn
```

Comment:
This comment refers to lines 111-175 - I think this section can be removed rather than being commented out, as it still exists in git history

Reply:
Removed.